stackstate_checks_base/tests/test_http_helper.py

```diff
@@ -0,0 +1,389 @@
+# coding=utf-8
+import unittest
+
+from schematics.models import Model, DataError
+from schematics.types import StringType, IntType, BooleanType
+
+from stackstate_checks.base.utils.common import read_file, load_json_from_file
+from stackstate_checks.utils.http_helper import (HTTPHelper, HTTPRequestType, HTTPAuthenticationType, HTTPResponseType)
+from stackstate_checks.utils.http_helper import HTTPHelperConnectionHandler, \
+    HTTPHelperRequestModel, HTTPHelperConnectionModel, HTTPHelperSessionModel
+from stackstate_checks.utils.http_helper import HTTPHelperRequestHandler, HTTPHelperSessionHandler
+
+"""
+Structures used within the test cases
+"""
+
+
+class BodySchematicTest(Model):
+    hello = IntType(required=True)
+
+
+class BodyRequestSchematicTest(Model):
+    hello = StringType(required=True)
+    pong = BooleanType(required=True)
+
+
+class BodyResponseSchematicTest(Model):
+    hello = StringType(required=True)
+    pong = BooleanType(required=True)
+
+
+class TestHTTPHelperBase(unittest.TestCase):
+    """
+     HTTP Helper Request Class
+    """
+
+    def test_compact_unicode_response(self):
```

Comment:
This test only proofs that you can deal with unicode characters in UTF-8, but this does not proof you can also deal with different unicode encodings.